### ISSUE TITLE
manylinux2010 fails with old pip 9

### ISSUE THREAD

**Issue by cbm755** [pypa/manylinux#389]
From glancing at #179 (second crossed out bullet point) I suspect this might be the intended behaviour, but my understanding here is low, so I'll file this just in case...

On Ubuntu 18.04, one has rather old pip:
```
$ /usr/bin/pip3 -V
pip 9.0.1 from /usr/lib/python3/dist-packages (python 3.6)
```
With this, I get
```
pip3 install -U PyMuPDF-1.16.7-cp36-cp36-manylinux2010_x86_64.whl
PyMuPDF-1.16.7-cp36-cp36-manylinux2010_x86_64.whl is not a supported wheel on this platform.
```
  * If I rename the file to `...-manylinux1_x86_64.whl` then it "works".
  * Or if upgrade my pip of course it works.

Is this intended?  Seems to me, Ubuntu 18.04 is going to be around as a docker-based and in VM contexts for quite some time...

**Comment by mayeut** [comment]
It's the expected behavior, support for manylinux2010 wheels has been added in `pip 19.0`
There are discussions on-going to mitigate this kind of issue in the future. c.f. https://www.python.org/dev/peps/pep-0600/ and the discourse thread linked in it.

**Comment by cbm755** [comment]
Ok.

manylinux2010 in pip 10 and manylinux2014 in pip 19?  Or did I misready #179 and #338?

**Comment by mayeut** [comment]
`pip 19.0` for manylinux2010, I corrected #179 (original planning was for 10.x but got shifted)
`pip 19.3` for manylinux2014

**Comment by cbm755** [comment]
thanks!

Its surprisingly non-trivial to upgrade pip3 on Ubuntu 18.04 in a way that works OOTB (say in a CI environment).  Here's what's I'm doing in my .gitlab-ci.yml file in case this helps someone who finds her way here:

```diff
 before_script:
   - apt-get update
   - DEBIAN_FRONTEND=noninteractive apt-get install -y tzdata curl
-  - pip3 install pymupdf
+  - pip3 install --upgrade pip
+  - which pip3
+  - python3 -m pip -V
+  - python3 -m pip install pymupdf
```
